NotificationServiceExtension/NotificationService.swift

```diff
@@ -67,6 +67,18 @@ class NotificationService: UNNotificationServiceExtension {
                     bestAttemptContent.body = fallbackPushContent
                 }
 
+                // Assigning interruption level and relevance score only available starting on iOS 15
+                if #available(iOS 15.0, *) {
+                    if finalNotifications.notificationsToDisplay.count == 1, let notification = finalNotifications.notificationsToDisplay.first {
+                        let priority = RemoteNotification.typeFrom(notification: notification).priority
+                        bestAttemptContent.interruptionLevel = priority.interruptionLevel
+                        bestAttemptContent.relevanceScore = priority.relevanceScore
+                    } else {
+                        bestAttemptContent.interruptionLevel = RemoteNotificationType.bulkPriority.interruptionLevel
```

Comment:
There is a chance that there are multiple notifications that all come from the same talk page (this is caught within the conditional above on line 61), in which case it could get more of a bulk talk page message treatment ("N new messages on {title})". These types would get the lower bulkPriority of .passive/0.2, but I think they should match the "Multiple messages" row in the spreadsheet. Maybe within here add something like this?

```
if NotificationServiceHelper.allNotificationsAreForSameTalkPage(notifications: finalNotifications.notificationsToDisplay) {
    bestAttemptContent.interruptionLevel = RemoteNotificationType.bulkTalkPriority.interruptionLevel //.timeSensitive
    bestAttemptContent. relevanceScore = RemoteNotificationType.bulkTalkPriority. relevanceScore //0.9
} else {
    bestAttemptContent.interruptionLevel = RemoteNotificationType.bulkPriority.interruptionLevel //passive
    bestAttemptContent. relevanceScore = RemoteNotificationType.bulkPriority.relevanceScore //0.2
}
```

Reply:
Whoops, I missed that sorry. Rationale to treat them with same priority as one makes sense – updated!